packages/utilities/src/utilities/settings/settings.js

```diff
@@ -11,11 +11,11 @@
  * @exports ibmdotcom.settings
  * @type {object} Settings object
  * @property {string} [prefix=dds]
- * Carbon for IBM.com v1.19.0',
+ * Carbon for IBM.com v1.19.1',
  *
  */
 const settings = {
-  version: 'Carbon for IBM.com v1.19.0',
+  version: 'Carbon for IBM.com v1.19.1',
```

Comment:
```suggestion
  version: 'Carbon for IBM.com v1.19.0',
```

Looks like this was based on the `v1.19.1` patch release branch.